torch/optim/optimizer.py

```diff
@@ -118,6 +118,22 @@ def _dispatch_sqrt(
         return math.sqrt(x)
 
 
+def _disable_dynamo_if_unsupported(func):
+    def maybe_fallback(self, *args, **kwargs):
+        if is_compiling() and not kwargs.get("capturable", False):
+            import torch._dynamo
```

Comment:
is this import still needed if you use @torch._disable_dynamo below?

Reply:
Nope I was playing around with something else I think, let me remove.